src/cli/abstraction/mod.rs

```diff
@@ -4,14 +4,25 @@ use clap::Args;
 pub(crate) struct AuthArguments {
 	#[arg(help = "Database authentication username to use when connecting")]
 	#[arg(env = "SURREAL_USER", short = 'u', long = "username", visible_alias = "user")]
-	#[arg(default_value = "root")]
 	pub(crate) username: String,
 	#[arg(help = "Database authentication password to use when connecting")]
-	#[arg(short = 'p', long = "password", visible_alias = "pass")]
-	#[arg(env = "SURREAL_PASS", default_value = "root")]
+	#[arg(short = 'p', long = "password", visible_alias = "pass", requires = "username")]
+	#[arg(env = "SURREAL_PASS")]
 	pub(crate) password: String,
```

Comment:
Here I remove the default user=root, pass=root arguments for two reasons:
- there should be no incentive i.e. ease of passing arguments to use weak passwords
- if the user/password default to root/root, then it's not possible to connect as an unauthenticated user